app/src/main/java/org/fossasia/openevent/app/module/event/list/EventListFragment.java

```diff
@@ -93,6 +100,27 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         return binding.getRoot();
     }
 
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        super.onCreateOptionsMenu(menu, inflater);
+        inflater.inflate(R.menu.menu_events, menu);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.sortByEventName:
+                initialized = false;
+                setupRecyclerView(SORTBYNAME);
```

Comment:
We don't want to reinitialize the whole adapter for changing sort strategy. Use fastadapter for it

Reply:
Sorry, i didn't get what you are trying to say, should i create a new adapter, can you please point me in the right direction

Reply:
Look at how to sort items in FastAdapter docs

Reply:
But event list is not generated through fast adapter, it uses RecyclerView Adapter

Reply:
OK, then sort in presenter, and set the new list in adapter and notify data set changed